src/NUnitTestAdapter/NUnit3TestDiscoverer.cs

```diff
@@ -61,10 +61,7 @@ public void DiscoverTests(IEnumerable<string> sources, IDiscoveryContext discove
             if (!Debugger.IsAttached)
                 Debugger.Launch();
 #endif
-            Initialize(discoveryContext, messageLogger);
-
-
-
+            Initialize(discoveryContext,messageLogger);
```

Comment:
Normally there's a space after the comma. Do you think you'd like using an analyzer that gives a squiggle and Ctrl+. fix so that you can spend minimal energy on this stuff? (Or would you rather not care about stuff like this? I don't want to annoy you!)

Reply:
Shouldn't this be in the editorconfig file ?

Reply:
It is the default, but there is no squiggly lines from that one, so yes, what analyzer do you suggest?

Reply:
Yes, but it can only be set to true or false, not `true:error`. That means that it'll follow the preference when you type `;` at the end of the line and it autoformats, but it won't show a warning or a Ctrl+. fix option. What do you think about that?

https://docs.microsoft.com/en-us/visualstudio/ide/editorconfig-formatting-conventions?view=vs-2019#csharp_space_after_comma

Reply:
Well, it works when you autoformat, but as you see, it (or I) doesn't always end up with the right result.  Squiggly lines are nice in those cases .

Reply:
I'd be happy to submit a PR adding an analyzer for this and you could see how you liked it. I use StyleCop and turn off at least half the rules. The remaining squiggles and code fixes make life easier for me.

Reply:
Please do :-)